sonar-update-center-mojo/src/main/java/org/sonar/updatecenter/mojo/HttpDownloader.java

```diff
@@ -39,7 +39,7 @@ public HttpDownloader(File outputDir, Log log) {
     this.log = log;
   }
 
-  public File download(String url, boolean force) throws IOException, URISyntaxException {
+  public File download(String url, boolean force, boolean verifyUrlIfCached) throws IOException, URISyntaxException {
```

Comment:
For a give HttpDownload, the argument verifyUrlIfCached has always the same value. It should be moved to a member field of the class HttpDownloader